docker/ichiran-scripts/ichiran-sbcl

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-sbcl --core /root/ichiran.core
\ No newline at end of file
+sleep 0.1 && rlwrap sbcl --core /root/ichiran.core --eval "(sb-ext:enable-debugger)"
```

Comment:
Is the sleep command intentionally left there?

Reply:
Yes, otherwise the terminal doesn't initialize in time and rlwrap errors. This is [one  solution](https://zgljl2012.com/rlwrap-error-my-terminal-reports-width-0-is-it-emacs-i-cant-handle-this-sorry/) I found that actually worked. It's quite ugly though...